kahuna/public/js/components/gr-add-keyword/gr-add-keyword.js

```diff
@@ -0,0 +1,129 @@
+import angular from 'angular';
+
+import '../../services/image-accessor';
+import '../../edits/service';
+import '../../forms/datalist';
+
+import './gr-add-keyword.css';
+import template from './gr-add-keyword.html';
+
+import '../../directives/gr-auto-focus';
+import {overwrite} from "../../util/constants/editOptions";
+
+export var addKeyword = angular.module('gr.addKeyword', [
+  'gr.image.service',
+  'kahuna.edits.service',
+  'gr.autoFocus',
+  'kahuna.forms.datalist'
+]);
+
+addKeyword.controller('GrAddKeywordCtrl', [
+  '$window', '$scope', '$q', 'imageAccessor', 'editsService',
+  function ($window, $scope, $q, imageAccessor, editsService) {
+
+    let ctrl = this;
```

Comment:
not re-assigned so needn't be a `let`
```suggestion
    const ctrl = this;
```
it is mutated too, but that's angular's way 😢